fern/definition/medical/document.yml

```diff
@@ -177,16 +214,79 @@ types:
+              contained:
+                - resourceType: Organization
+                  id: org522
+                  name: Hospital 522
+                - resourceType: Practitioner
+                  id: auth925
+                  name:
+                    - text: John Snow
+                      family: Snow
+                      given: 
+                        - John
+              extension:
+                - url: "https://public.metriport.com/fhir/StructureDefinition/data-source.json"
+                  valueCoding:
+                    system: "https://public.metriport.com/fhir/StructureDefinition/data-source.json"
+                    code: METRIPORT
+              masterIdentifier:
+                system: urn:ietf:rfc:3986
+                value: 018a875a-3181-74e8-8aa4-40c59ed7e1b2
+              identifier:
+                - use: official
+                  system: urn:ietf:rfc:3986
+                  value: 018a875a-3181-74e8-8aa4-40c59ed7e1b2
+              status: current
+              type:
+                coding:
+                  - system: http://loinc.org/
+                    code: 34133-9
+                    display: Summarization of episode note
+              subject:
+                reference: Patient/018a5164-c0ac-79e2-99cc-9cf951c26d94
+                type: Patient
+              date: "2023-08-15T03:05:58.090Z"
+              author:
+                - reference: "#org522"
+                  type: Organization
+              description: C-CDA R2.1 Patient Record
+              content:
+                - extension:
+                    - url: "https://public.metriport.com/fhir/StructureDefinition/data-source.json"
+                      valueCoding:
+                        system: "https://public.metriport.com/fhir/StructureDefinition/data-source.json"
+                        code: METRIPORT
+                  attachment:
+                    contentType: application/pdf
+                    url: "http://api.metriport.com/doc-contribution/commonwell/?fileName=965f9957-34ac-442b-9a17-fa09c76c16da/018a5164-c0ac-79e2-99cc-9cf951c26d94/965f9957-34ac-442b-9a17-fa09c76c16da_018a5164-c0ac-79e2-99cc-9cf951c26d94_018a875a-3181-74e8-8aa4-40c59ed7e1b2"
+                    size: 1080416
+                    title: 965f9957-34ac-442b-9a17-fa09c76c16da/018a5164-c0ac-79e2-99cc-9cf951c26d94/965f9957-34ac-442b-9a17-fa09c76c16da_018a5164-c0ac-79e2-99cc-9cf951c26d94_018a875a-3181-74e8-8aa4-40c59ed7e1b2
+                    creation: "2023-08-14T22:05:58-05:00"
+                  format:
+                    code: urn:ihe:pcc:xphr:2007
+              context:
+                period:
+                  start: "2023-08-15T02:05:58.090Z"
+                  end: "2023-08-15T03:05:58.090Z"
+                sourcePatientInfo:
+                  reference: Patient/018a5164-c0ac-79e2-99cc-9cf951c26d94
+                  type: Patient
 
-  DocumentReference:
+  DocumentReferenceDocumentWebhook:
```

Comment:
We only use this schema in the return for DocumentQueries in the webhook. Unless we are going ti implement any validation of webhook responses (I don't think thats part of the sdk as of now but maybe it should be?) then this should all be deleted